### ISSUE TITLE
Performance issue connecting to MSSQL

### ISSUE THREAD

**Issue by robarchibald** [denisenkom/go-mssqldb#67]
Thanks for the nice library.  It's great to have access to SQL Server on Go!

I wrote a very simple client which connects to my local SQL Server 2012 instance and does a very simple select statement.  I then wrote a similar ASP.Net Web API service and compared the performance of each using JMeter just doing a whole bunch of simultaneous get requests.  Both saturate the CPU, but they look very different in how they use resources which is weird.  I was very surprised to see that the Go process uses up 340% of the CPU resources on SQL Server that the ASP.Net Web API server does.  Thinking that Web API must be doing some optimization, I added a trace in SQL Server and found that both are running the same number of queries.  And, the queries are the same between the two servers.  I was also very confused about the "Local Security Authority Process" which is using 12.7% of the Go server and nothing with Web API.  If I remove the SQL query, LSASS disappears from the list.  Having never heard of this process before, I looked it up on Wikipedia and found that: "(LSASS), is a process in Microsoft Windows operating systems that is responsible for enforcing the security policy on the system. It verifies users logging on to a Windows computer or server, handles password changes, and creates access tokens."  Are you doing some policy enforcement on every query or something like that?  Thinking it had something to do with encryption, I tried to change the settings for encryption in the connection string, and then it failed every time.  I'll log a separate bug for that issue.  

In summary:
1. Why would the go driver which is doing the same thing as .Net Web API consume 340% of the SQL Server CPU resources?
2. Why is the Local Security Authority Process using so much CPU with the Go REST server and nothing with ASP.Net Web API?

Here's a snapshot of it running the Go REST server:
![image](https://cloud.githubusercontent.com/assets/10640088/5846416/4a924c86-a178-11e4-9ce4-c725cc410d39.png)

Here's a snapshot of it running the IIS Web API server:
![image](https://cloud.githubusercontent.com/assets/10640088/5846381/ff37417e-a177-11e4-8ee0-d88b135793fd.png)

**Comment by denisenkom** [comment]
Are you using the same authentication method and encryption settings in go and .Net?

**Comment by robarchibald** [comment]
Yes, I was using the same mechanism for both.  Since opening this issue, I've now left the company where I was using this, so I don't have any further insight on this.  I'm going to go ahead and close.
